src/CoreWCF.Http/tests/ClientContract/IXmlSerializerContract.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.ServiceModel;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace ClientContract
+{
+	[AttributeUsage(AttributeTargets.All, Inherited = false)]
+	internal sealed class __DynamicallyInvokableAttribute : Attribute
```

Comment:
The __DynamicallyInvokableAttribute attribute was used for some Windows 8 mechanism for .NET Framework being used in a Windows App Store application. It's not needed so remove this attribute and all references to it.

Reply:
Okay, I have deleted this attribute and all references to it.